intelmq/bots/parsers/shadowserver/README.md

```diff
@@ -61,4 +61,11 @@ Every bot-type is defined by a dictionary with three values:
   tuple of showserver key and conversion function or not mention it at all.
   In both cases, the data will be added to extra.
 - `additional_fields`: A dictionary with a static mapping of field name to
-  data, e.g. to set classifications or protocols.
\ No newline at end of file
+  data, e.g. to set classifications or protocols.
+
+The tuples can be of following format:
+
+- `('intelmqkey', 'shadowkey')`
+- `('intelmqkey', 'shadowkey', conversion_function)`
+- `('shadowkey', conversion_function)`, data will be added to extra in this case
```

Comment:
I have the feeling, that this behaviour is not clear enough.
What about an `extra_fields` dict?